charts/pythie/templates/pythie-serving-deployment.yaml

```diff
@@ -103,6 +103,10 @@ spec:
             {{- end }}
             - name: "VERBOSITY_LEVEL"
               value: "0"
+            {{- if $.Values.pythieServing.opentelemetryCollectorHost }}
```

Comment:
Why `$.Values.pythieServing.opentelemetryCollectorHost` and not `$servingValue.opentelemetryCollectorHost` ?